### DISCUSSION TITLE
What directory is files relating to the movableAmethyst located in?

### DISCUSSION THREAD

**Discussion by Elmartino4** [gnembon/fabric-carpet#972] [Q&A] [answered]
I'm intending on modifying this particular setting slightly so that it is more challenging to transport this block than just mining it with silk touch, but I'm struggling to find any files relating to this setting.
Thanks in advance for any help!

**Comment by altrisi** [comment, marked as answer]
Both the movable behaviour and the silk touch part are defined in `BuddingAmethystBlock_movableAmethystMixin`, in the 1.17 branch (master is targeting 1.16).

**Reply by Elmartino4** [reply]
Thanks, found it (I was looking in the master branch)
